iocore/eventsystem/I_EventProcessor.h

```diff
@@ -307,6 +306,38 @@ class EventProcessor : public Processor
   volatile int n_dthreads       = 0; // No. of dedicated threads
   volatile int thread_data_used = 0;
   ink_mutex dedicated_spawn_thread_mutex;
+
+  /// Provide container style access to just the active threads, not the entire array.
+  struct active_threads_type {
```

Comment:
Since this struct is an implementation detail, can it be private? Or it needs to be exposed for `active_Xthreads()` to work?